### PR TITLE
Add "recent files" icon to navigation

### PR DESCRIPTION
**Pull request by Bugsbane**
Adds the recent files navigation icon (PNG and SVG) and the class to include it in the files app sidebar navigation.

### CODE REVIEW

**Comment by williambargent** [comment]
:+1:

**Comment by MorrisJobke** [comment]
@Bugsbane Could I ask you to remove the PNG. It is not needed anymore. Master will not support IE8 and 9. That are the only ones where the PNG was needed :) 

Beside that 👍

File: apps/files/css/files.css [hunk 1]
```diff
@@ -86,6 +86,9 @@
 }
 
 /* icons for sidebar */
+.nav-icon-recent {
+	background-image: url('../img/recent.svg');
+}
```
Comment (jancborchardt) [inline apps/files/css/files.css]: Please put the CSS in order as it is in the sidebar. :) That is, first nav-icon-files, and then nav-icon-recent.

**Comment by LukasReschke** [comment]
Closing until I helped rebasing later. See my message in the forum.

**Pull request closed without merging**